src/Sylius/Bundle/OrderBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -52,15 +52,4 @@ public function isNumberUsed($number)
             ->getSingleScalarResult() > 0
         ;
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    protected function getQueryBuilder()
```

Comment:
This was usefull if someone extends this repositore & don't want to always declare that join.